src/docs/product/performance/index.mdx

```diff
@@ -17,6 +17,32 @@ The Performance Monitoring homepage is the main view where you can search or bro
 
 The search bar operates similarly to the one in the Discover [Query Builder](/product/discover-queries/query-builder/#filter-by-search-conditions). For example, you can refine your transaction search to a specific release by using `release:` as a key field and assigning the version as a value.
 
+## Configure Display
+
+The "Display" dropdown automatically selects a set of [metrics](metrics/) and table columns based on the platform of the selected project. The "Frontend (Pageload)" and "Frontend (Other)" options additionally filter by pageload and non-pageload transactions.
+
+### Display: All
+
+In Display: All mode, Apdex and TPM are shown. The Apdex threshold can be configured in the project settings.
+
+![Performance Homepage with Display: All](00_performance_landing.png)
+
+### Display: Frontend (Pageload)
+
+In Display: Frontend (Pageload) mode, [web vitals](web-vitals/) and LCP information are shown. Additionally, transactions are automatically filtered by `transaction.op:pageload`. This generally corresponds to all browser page transitions that are not handled internally in SPA frameworks such as React or Vue.
+
+![Performance Homepage with Display: Frontend (Pageload)](01_performance_landing.png)
+
+### Display: Frontend (Other)
+
+In Display: Frontend (Other) mode, duration information is shown. Additionally, transactions are automatically filtered by `transaction.op:!pageload`. This generally corresponds to page transitions handled internally in SPA frameworks such as React or Vue. Currently, this will also display backend transactions if a backend project is selected.
```

Comment:
Instead of writing `Currently, this will also display backend transactions if a backend project is selected.` here, perhaps in Configure display we mention that we infer a display from the project's platform but can choose whichever display you prefer for your selected project, including a display that might not apply?

Reply:
what I want to communicate here is that if you select multiple projects, you may expect "Display: Frontend (Other)" to only select frontend transactions, but in fact it will show both frontend and backend transactions as long as they aren't pageloads.

Reply:
I adjusted this text in v2, please re-review :)